samples/apps/android/LwcTestApp/app/src/main/java/com/salesforce/mobiletooling/lwctestapp/MainActivity.java

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright (c) 2020, salesforce.com, inc. All rights reserved.
+SPDX-License-Identifier: MIT
+For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+*/
+
+package com.salesforce.mobiletooling.lwctestapp;
```

Comment:
Not sure how I feel about starting a new Android project in Java. I think it would be better to use Kotlin.

Reply:
oh ok sure, will convert to Kotlin.